index.js

```diff
@@ -0,0 +1,106 @@
+import './style.css';
+import TodoList from './modules/tasks';
+import toggleStatus from './modules/clear';
+
+let todolist = [];
+if (JSON.parse(localStorage.getItem('todolist'))) {
+  todolist = JSON.parse(localStorage.getItem('todolist')).todolist;
+}
+const newTodoList = new TodoList(todolist);
+const todoItems = document.getElementsByClassName('to-do-list')[0];
+
+const sortedTodoList = todolist.sort((a, b) => a.index - b.index);
+
+sortedTodoList.forEach((todo) => {
+  const task = document.createElement('li');
+  // task.classList.add('task');
+  task.id = todo.index;
+  task.innerHTML = `<span class="task"><input type="checkbox" name="${todo.index}" class="check">
+  <label class = "${todo.index} task-desc black" for="${todo.index}">${todo.description}</label>
+  </span>
+  <div class="remove-button">
+    <i class='fa fa-trash ash'></i>
+  <div>`;
+  todoItems.appendChild(task);
+});
+
+const enter = document.querySelector('.enter');
+
+const addTask = (e) => {
+  e.preventDefault();
+  const newItem = document.getElementById('data');
+  if (newItem.value) {
+    const description = newItem.value.trim();
+    const index = todolist.length + 1;
+    newTodoList.addTask(description, index);
+    localStorage.setItem('todolist', JSON.stringify(newTodoList));
+    newItem.value = '';
+  }
+  window.location.reload();
```

Comment:
Please Kindly avoid using `window` to reach the `location` method, You can use `location` directly like this: `location.reload();`